### ISSUE TITLE
Error with sidechain dihedrals persists

### ISSUE THREAD

**Issue by patwin2** [AG-Peter/encodermap#26]
So I tried

"pip install --upgrade git+https://github.com/AG-Peter/encodermap.git"

as suggested, but I still get the same error message (I updated pip prior to trying).

For reference, here is the yml file for the environment I'm working in. Are any package versions obviously wrong?

---------------------------------------------------------------------------------------------------------

name: encodermap
channels:

conda-forge
defaults
dependencies:
_libgcc_mutex=0.1=conda_forge
_openmp_mutex=4.5=2_gnu
bzip2=1.0.8=h7f98852_4
ca-certificates=2023.5.7=hbcca054_0
cudatoolkit=11.4.2=h7a5bcfd_11
cudnn=8.2.1.32=h86fa8c9_0
ld_impl_linux-64=2.40=h41732ed_0
libffi=3.4.2=h7f98852_5
libgcc-ng=12.2.0=h65d4601_19
libgomp=12.2.0=h65d4601_19
libnsl=2.0.0=h7f98852_0
libsqlite=3.42.0=h2797004_0
libstdcxx-ng=12.2.0=h46fd767_19
libuuid=2.38.1=h0b41bf4_0
libzlib=1.2.13=h166bdaf_4
ncurses=6.3=h27087fc_1
openssl=3.1.0=hd590300_3
python=3.9.16=h2782a2a_0_cpython
readline=8.2=h8228510_1
setuptools=67.7.2=pyhd8ed1ab_0
tk=8.6.12=h27826a3_0
wheel=0.40.0=pyhd8ed1ab_0
xz=5.2.6=h166bdaf_0
pip:
absl-py==1.4.0
anyio==3.6.2
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
arrow==1.2.3
asttokens==2.2.1
astunparse==1.6.3
attrs==23.1.0
backcall==0.2.0
beautifulsoup4==4.12.2
biopython==1.81
bleach==6.0.0
cachetools==5.3.0
certifi==2023.5.7
cffi==1.15.1
charset-normalizer==3.1.0
click==8.1.3
cloudpickle==2.2.1
comm==0.1.3
contourpy==1.0.7
cycler==0.11.0
dask==2023.5.0
debugpy==1.6.7
decorator==5.1.1
deeptime==0.4.4
defusedxml==0.7.1
dill==0.3.6
distributed==2023.5.0
encodermap==3.0.0
executing==1.2.0
fast-histogram==0.11
fasteners==0.18
fastjsonschema==2.16.3
flatbuffers==23.5.9
fonttools==4.39.4
fqdn==1.5.1
fsspec==2023.5.0
gast==0.4.0
google-auth==2.18.1
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
griddataformats==1.0.1
grpcio==1.54.2
gsd==2.8.1
h5netcdf==1.1.0
h5py==3.8.0
humanfriendly==10.0
idna==3.4
importlib-metadata==6.6.0
importlib-resources==5.12.0
ipykernel==6.23.1
ipython==8.13.2
ipython-genutils==0.2.0
ipywidgets==7.7.5
isoduration==20.11.0
jedi==0.18.2
jinja2==3.1.2
joblib==1.2.0
jsonpointer==2.3
jsonschema==4.17.3
jupyter-client==8.2.0
jupyter-core==5.3.0
jupyter-events==0.6.3
jupyter-server==2.5.0
jupyter-server-terminals==0.4.4
jupyterlab-pygments==0.2.2
jupyterlab-widgets==1.1.4
keras==2.10.0
keras-preprocessing==1.1.2
kiwisolver==1.4.4
libclang==16.0.0
llvmlite==0.40.0
locket==1.0.0
markdown==3.4.3
markupsafe==2.1.2
matplotlib==3.7.1
matplotlib-inline==0.1.6
mdanalysis==2.4.3
mdshare==0.4.2
mdtraj==1.9.7
mistune==2.0.5
mmtf-python==1.1.3
mpl-scatter-density==0.7
mrcfile==1.4.3
msgpack==1.0.5
multiprocess==0.70.14
nbclassic==1.0.0
nbclient==0.7.4
nbconvert==7.4.0
nbformat==5.8.0
nest-asyncio==1.5.6
networkx==3.1
nglview==3.0.4
notebook==6.5.4
notebook-shim==0.2.3
numba==0.57.0
numpy==1.24.3
oauthlib==3.2.2
opt-einsum==3.3.0
packaging==23.1
pandas==2.0.1
pandocfilters==1.5.0
parso==0.8.3
partd==1.4.0
pathos==0.3.0
pexpect==4.8.0
pickleshare==0.7.5
pillow==9.3.0
pip==23.2.1
platformdirs==3.5.1
pox==0.3.2
ppft==1.7.6.6
prometheus-client==0.16.0
prompt-toolkit==3.0.38
protobuf==3.19.6
psutil==5.9.5
ptyprocess==0.7.0
pure-eval==0.2.2
pyasn1==0.5.0
pyasn1-modules==0.3.0
pycparser==2.21
pyemma==2.5.12
pygments==2.15.1
pyparsing==3.0.9
pyrsistent==0.19.3
python-dateutil==2.8.2
python-json-logger==2.0.7
pytz==2023.3
pyyaml==6.0
pyzmq==25.0.2
requests==2.30.0
requests-oauthlib==1.3.1
rfc3339-validator==0.1.4
rfc3986-validator==0.1.1
rsa==4.9
scikit-learn==1.2.2
scipy==1.10.1
seaborn==0.12.2
send2trash==1.8.2
six==1.16.0
sniffio==1.3.0
sortedcontainers==2.4.0
soupsieve==2.4.1
stack-data==0.6.2
tblib==1.7.0
tensorboard==2.10.1
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.10.0
tensorflow-estimator==2.10.0
tensorflow-io-gcs-filesystem==0.32.0
termcolor==2.3.0
terminado==0.17.1
threadpoolctl==3.1.0
tinycss2==1.2.1
toolz==0.12.0
tornado==6.3.2
tqdm==4.65.0
traitlets==5.9.0
typing-extensions==4.5.0
tzdata==2023.3
uri-template==1.2.0
urllib3==1.26.15
wcwidth==0.2.6
webcolors==1.13
webencodings==0.5.1
websocket-client==1.5.1
werkzeug==2.3.4
widgetsnbextension==3.6.4
wrapt==1.15.0
xarray==2023.4.2
zict==3.0.0
zipp==3.15.0

------------------------------------------------------------------------------------------------

Thanks.

**Comment by kevinsawade** [comment]
It should now be fixed.
